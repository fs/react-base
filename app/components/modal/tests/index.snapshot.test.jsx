import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Modal from '../';

describe('Modal', () => {
  let props;
  const renderComponent = () => shallow(<Modal {...props} />);

  beforeEach(() => {
    props = {
      closeModal: () => {},
      isOpen: true,
      title: 'Test Modal',
    };
  });

  it('renders correctly', () => {
    expect(toJson(renderComponent())).toMatchSnapshot();
  });
});
