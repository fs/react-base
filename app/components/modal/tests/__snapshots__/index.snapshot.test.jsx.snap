// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders correctly 1`] = `
<Modal
  animation={true}
  autoFocus={true}
  backdrop={true}
  bsClass="modal"
  bsSize="small"
  dialogComponentClass={[Function]}
  enforceFocus={true}
  keyboard={true}
  manager={
    ModalManager {
      "add": [Function],
      "containers": Array [],
      "data": Array [],
      "handleContainerOverflow": true,
      "hideSiblingNodes": true,
      "isTopModal": [Function],
      "modals": Array [],
      "remove": [Function],
    }
  }
  onHide={[Function]}
  renderBackdrop={[Function]}
  restoreFocus={true}
  show={true}
>
  <ModalHeader
    bsClass="modal-header"
    closeButton={true}
    closeLabel="Close"
  >
    <h3
      className="modal-title"
    >
      Test Modal
    </h3>
  </ModalHeader>
  <ModalBody
    bsClass="modal-body"
    componentClass="div"
  />
</Modal>
`;
